Automattic/studio
# Studio: Add Sync screen for when there are no sites connected

Resolves https://github.com/Automattic/dotcom-forge/issues/9353

## Proposed Changes

This PR adds the UI for the case when where are no connected sites under the `Sync` tab. it does not implement any functionality for `Connect site` and `Create a site` buttons (e.g. clicking the buttons does not do anything). This is only the implementation of the UI part. 

## Testing Instructions



* Pull the changes from this branch
* Start the app with `STUDIO_SITE_SYNC=true npm start`
* Navigate to the `Sync` tab
* Observe the screen that is indicated below:

<img width="978" alt="Screenshot 2024-10-11 at 1 50 01 PM" src="https://github.com/user-attachments/assets/07c0d9ca-271a-4834-8d0c-af8b475de516">

* Log out of your account
* Observe that the screen is now changed to:

<img width="1011" alt="Screenshot 2024-10-11 at 1 50 17 PM" src="https://github.com/user-attachments/assets/c0cb2039-98e6-4a9c-b5cf-b97c35f1e5a1">

* Log back into your account
* Disable your WiFi or internet connection
* Observe that the screen changes to the following:

<img width="1090" alt="Screenshot 2024-10-11 at 2 02 18 PM" src="https://github.com/user-attachments/assets/d9650a29-d1c0-4b60-93bf-8d0e0abe995a">

* Confirm that the buttons are disabled and that when you hover over them, you can see the tooltip explaining that you need an internet connection. 

Figma: RToz6tIuQ7nlZrikBte4GU-fi-4491_115759

## Pre-merge Checklist
